app/build.gradle

```diff
@@ -91,13 +91,13 @@ android {
         }
     }
 
-    variantFilter { variant ->
-        def flavors = variant.flavors*.name
-        // We only need a gecko debug and beta build for now.
-        if (flavors.contains("gecko") && ((variant.buildType.name != "debug" && variant.buildType.name != "beta") || flavors.contains("klar"))) {
-            setIgnore(true)
-        }
-    }
+//    variantFilter { variant ->
+//        def flavors = variant.flavors*.name
+//        // We only need a gecko debug and beta build for now.
+//        if (flavors.contains("gecko") && ((variant.buildType.name != "debug" && variant.buildType.name != "beta"))) {
+//            setIgnore(true)
+//        }
+//    }
```

Comment:
We should update this. I guess we can now change the code to build all Focus+WebView and Klar+GeckoView combinations but never Focus+GeckoView or Klar+WebView.